### ISSUE TITLE
(3.0.53) a third and fourth `test_addedProjectNotOnDisk` *Service* test failing

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#366]
The first two failing tests reported in Issue #333. Now a third and fourth tests are failing (`RowanTestClassServiceTest>>#test_addedProjectNotOnDisk`):
```
Rowan STON Cypress Tonel tests for GemStone '3.2.15'
643 run, 633 passed, 5 failed, 5 errors
  errors
	RwBrowserToolApiTest>>#testMoveGlobalExtensionSessionMethods
	RwBrowserToolApiTest>>#testNewClassVersion_session_method_change_extension_method_source
	RwHybridBrowserToolTest>>#testHybridMoveMethodFromSessionMethodsIntoSessionMethods
	RwHybridBrowserToolTest>>#testHybridMoveMethodIntoSessionMethods
-->	RwRowanProjectIssuesTest>>#testIssue150_branches
  failures
-->	RowanClassServiceTest>>#test_addedProjectNotOnDisk
-->	RowanProjectServiceTest>>#test_addedProjectNotOnDisk
-->	RowanTestClassServiceTest>>#test_addedProjectNotOnDisk
	RwLoadingTest>>#testPoolDictionaryChanges
	RwSymbolDictionaryTest>>#testClassVersioningPatch
```
I have a feeling that these tests must not be failing when run interactively. The original bug was reported 3-4 weeks ago ... marking `P1-critical', so that the bugs can be fixed when run in batch mode, which is how I validate releases ... if these tests are meant to be failing, then that's fine, but I should be told, so that I don't have to keep reporting newly failing tests in this vein...

**Comment by ericwinger** [comment]
``RwRowanProjectIssuesTest>>#testIssue150_branches`` is newly introduced. 

The other 3 are expected to fail (today).

**Comment by ericwinger** [comment]
Fix for ``RwRowanProjectIssuesTest>>#testIssue150_branches `` in https://github.com/GemTalk/Rowan/commit/74b9ffc818060c57315e8d559a1911ddd356b86e

**Comment by ericwinger** [comment]
-->	RowanClassServiceTest>>#test_addedProjectNotOnDisk
-->	RowanProjectServiceTest>>#test_addedProjectNotOnDisk
-->	RowanTestClassServiceTest>>#test_addedProjectNotOnDisk

Fixed in Oscar 3.0.56 on the server. https://github.com/GemTalk/Rowan/commit/e22468cbc873a87f72f90ab459bb13500774c556
